### ISSUE TITLE
'Set' not required by default

### ISSUE THREAD

**Issue by jimmynguyc** [edisonywh/behaves#10]
```ruby
irb(main):001:0> require 'behaves'
=> true
irb(main):002:0> class Animal
irb(main):003:1>   extend Behaves
irb(main):004:1> 
irb(main):005:1>   implements :speak, :eat
irb(main):006:1> end
NameError: uninitialized constant Behaves::Set
	from /Users/jimmy/.rbenv/versions/2.3.3/lib/ruby/gems/2.3.0/gems/behaves-0.1.0/lib/behaves.rb:5:in `implements'
	from (irb):6:in `<class:Animal>'
	from (irb):6
	from /Users/jimmy/.rbenv/versions/2.3.3/bin/irb:11:in `<main>'
```

**Comment by jimmynguyc** [comment]
Ahh it's been fixed on https://github.com/edisonywh/behaves/commit/df0d8c8529f76d4c57a3068309238af1a388cc33#diff-e695a3c248ff6aaefa123b6fd461e45f

**Comment by edisonywh** [comment]
Hey! Thanks for the issue Jimmy. You're right I had fixed it, but I have not yet re-published it, that's why you're still seeing the error. Apologies for that!

I've pushed a fix and it should be under `0.1.1` on RubyGems now!
